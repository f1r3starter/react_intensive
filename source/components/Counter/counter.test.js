// Core
import React from 'react';
import Counter from './';
import renderer from 'react-test-renderer';

const renderTree = renderer.create(<Counter count = { 3 } />).toJSON();

test('Counter component should be equal to its snapshot', () => {
    expect(renderTree).toMatchSnapshot();
});
