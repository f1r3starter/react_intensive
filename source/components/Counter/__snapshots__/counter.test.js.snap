// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counter component should be equal to its snapshot 1`] = `
<section
  className="counter"
>
  Posts count: 
  3
</section>
`;
